CONTRIBUTING.md

```diff
@@ -2,71 +2,79 @@
 
 ⚠️ 🌏 👀 First and foremost, remember that this repo is **open source**.
 
-Don't post anything or commit any code that isn't ready for the entire world to see. Avoid making specific reference to internal processes or features under development. While a lot of this information is probably harmless, it's better to be safe than sorry.
+Don't post anything or commit any code that isn't ready for the entire world to see.
+Avoid making specific reference to internal processes or features under development.
+While a lot of this information is probably harmless, it's better to be safe than sorry.
 
-If you work at SEEK and run into issues along the way, or even if you find some of these steps confusing or intimidating, please reach out to your friendly neighbourhood developer in the sku development Slack channel. We'll be super excited to help out!
+If you work at SEEK and run into issues along the way, or even if you find some of these steps confusing or intimidating, please reach out to your friendly neighbourhood developer `#sku-support` channel.
+We'd be happy to help out!
 
 ## Setup
 
-First, install [Node.js v8+](https://nodejs.org/).
+1. (Optional) Fork the repo if you do not have write access
+1. Clone the repo
+1. Install the appropriate versions of `pnpm` and `node` as specified in `package.json` and `.nvmrc`. This repo is configured to use [`volta`]
+   for managing toolchain dependencies, but feel free to use `nvm`, `corepack`, or whatever suits you best.
+1. Run `pnpm run test` to ensure everything is working as expected
 
-After cloning the project, install the dependencies:
+[`volta`]: https://volta.sh/
 
-```bash
-$ npm install
-```
-
-## Before Starting
-
-If you're planning to change the public API, please [open a new issue](https://github.com/seek-oss/sku/issues/new) and follow the provided RFC template. If you think it's a more straightforward API change and doesn't require a formal RFC, feel free to raise it in Slack first to see what others think.
-
-## Workflow
+## Making Changes
 
-Before starting your work, first ensure you're in the `master` branch and that you've pulled down the latest changes:
+Before starting your work, first ensure you've checked out the `master` branch and have pulled down the latest changes.
 
-```bash
-$ git checkout master
-$ git pull
+```sh
+git checkout master
+git pull
 ```
 
 Next, create a new branch for your work, with an appropriate name for your change:
 
-```bash
-$ git checkout -b add-my-cool-new-feature
+```sh
+git checkout -b add-my-cool-new-feature
 ```
 
+## Testing Your Changes
+
 To run the test suite locally:
 
-```bash
-$ npm test
+```sh
+pnpm run test
```

Comment:
Might be worth noting they'll need to run the setup-test-hosts script
```sh
sudo pnpm setup-test-hosts
```

Reply:
Good catch, forgot about that step.